providers/mysqlxorm/xorm.go

```diff
@@ -34,6 +35,9 @@ import (
 type Interface interface {
 	DB() *xorm.Engine
 	NewSession(ops ...SessionOption) *Session
+	Close() error
+	GetCloseState() bool
```

Comment:
it's unnecessary to expose `GetCloseState` method. The `Close` method can be invoked multiple times idempotently.

Reply:
Done.